### PR TITLE
Change text for key risk_details_information_body_period_logged

### PR DESCRIPTION
**Pull request by schauersbergern**
Change text for key risk_details_information_body_period_logged

### CODE REVIEW

**Review by KathrinLuecke — APPROVED** [review]
UA approved

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -300,7 +300,7 @@
     <string name="risk_details_subtitle_period_logged">"Dieser Zeitraum wird berücksichtigt."</string>
     <!-- XHED: risk details - infection period logged information body, below behaviors -->
     <!--    Dialog part 1-->
-    <string name="risk_details_information_body_period_logged">"Wenn Sie die Risiko-Ermittlung aktiviert haben, berechnet die App Ihr persönliches Infektionsrisiko. Als Grundlage dafür misst sie, über welche Dauer und mit welchem Abstand in den letzten %d Tagen Begegnungen mit Corona-positiv getesteten Personen stattfanden. Bei der Risikoermittlung werden außerdem Ihre Check-Ins berücksichtigt. Sie werden gewarnt, wenn Sie sich zur selben Zeit oder bis 30 Minuten nach der positiv getesteten Person im selben Raum aufgehalten haben."</string>
+    <string name="risk_details_information_body_period_logged">"Die Berechnung des Infektionsrisikos kann nur für die Zeiträume erfolgen, an denen die Risiko-Ermittlung aktiv war. Die Risiko-Ermittlung sollte daher dauerhaft aktiv sein. Für Ihre Risiko-Ermittlung wird der Zeitraum der letzten %d Tage betrachtet."</string>
```
Comment (Ein-Tim) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:303]:
```suggestion
    <string name="risk_details_information_body_period_logged">"Die Berechnung des Infektionsrisikos kann nur für die Zeiträume erfolgen, an denen die Risiko-Ermittlung aktiv war. Die Risiko-Ermittlung sollte daher dauerhaft aktiv sein. Für die Risiko-Ermittlung wird der Zeitraum der letzten %d Tage betrachtet."</string>
```

"Ihre Risiko-Ermittlung" makes no sense.

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Why is the part with the check-ins dropped? 

One in-line comment.

cc @kathrinLuecke

File: Corona-Warn-App/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (KathrinLuecke) [reply Corona-Warn-App/src/main/res/values-de/strings.xml:303]: This is the text that appears on the risk detail screen, for example if you get a warning about a low risk. It was always like this for i/OS, but had the wrong text for Android. With this PR the text is just aligned on Android.

Comment (Ein-Tim) [reply Corona-Warn-App/src/main/res/values-de/strings.xml:303]: @KathrinLuecke 
Ok, thanks for clarifying.

**Pull request merged**